elements/pfelement/src/pfelement.js

```diff
@@ -1,11 +1,31 @@
 import { autoReveal } from "./reveal.js";
-const prefix = "pfe-";
+import { isAllowedType, isValidDefaultType } from "./attrDefValidators.js";
+
+const tagPrefix = "pfe-";
 
 class PFElement extends HTMLElement {
   static create(pfe) {
+    pfe._validateProperties();
     window.customElements.define(pfe.tag, pfe);
   }
 
+  /**
+   * This prefix is prepended to the name of any attributes defined on the base
+   * class.  In other words, attributes that can be used on any element will be
+   * prefixed with this string.
+   *
+   * @example A "theme" prop, when reflected as an attribute, prefixed with "pfe-g-": <pfe-foo pfe-g-theme="bar">
+   */
+  static get globalAttrPrefix() {
+    return "pfe-g-";
+  }
+
+  static get attrPrefix() {
+    return "pfe-c-";
+  }
+
+  static get properties() {}
```

Comment:
Is this empty placeholder needed?  #justcurious

Reply:
No reason for it to be there yet. So we're going to leave it.